### PR TITLE
Prompt user to re-connect Ledger

### PR DESCRIPTION
**Pull request by Patrick1904**

### CODE REVIEW

File: src/utils/wallet.js [hunk 1]
```diff
@@ -415,7 +420,18 @@ class Wallet {
 
     async addLedgerAccessKey(accountId, ledgerPublicKey) {
         await setKeyMeta(ledgerPublicKey, { type: 'ledger' })
-        return await this.getAccount(accountId).addKey(ledgerPublicKey)
+        await this.getAccount(accountId).addKey(ledgerPublicKey)
+        await this.postSignedJson('/account/ledgerKeyAdded', { accountId, publicKey: ledgerPublicKey.toString() })
+    }
+
+    async connectLedger(ledgerPublicKey) {
+        const accessKeys =  await this.getAccessKeys()
```
Comment (vgrichina) [inline src/utils/wallet.js:428]: @Patrick1904 seems like we can just always call `setKeyMeta` in this case

Comment (vgrichina) [reply src/utils/wallet.js:428]: I also think this should potentially merged with `addLedgerAccessKey`, I'll take a look what it takes to consolidate

Comment (Patrick1904) [reply src/utils/wallet.js:428]: @vgrichina thanks let me know if you need me to make any changes

Comment (vgrichina) [reply src/utils/wallet.js:428]: @Patrick1904 I'll merge as is, let's try to rework in follow-up

I think probably makes sense to rework more globally: get rid of `setKeyMeta` as it is now, etc

**Pull request merged**